README.md

```diff
@@ -244,3 +243,64 @@ go test  # Run the tests
 ## Building Test Images
```

Comment:
This section about Building Test Images might be out of date.
@claudiubelu has been working very hard to move the building of the windows test images into kubernetes/kubernetes.
I don't think that transition is fully complete yet tho...

Reply:
filed this https://github.com/kubernetes-sigs/windows-testing/issues/225  so we can maybe iterate that in a next patch ?